src/main/java/ru/yandex/practicum/filmorate/service/DbFilmService.java

```diff
@@ -0,0 +1,87 @@
+package ru.yandex.practicum.filmorate.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.yandex.practicum.filmorate.dao.DaoFilmRepository;
+import ru.yandex.practicum.filmorate.dao.DaoLikeRepository;
+import ru.yandex.practicum.filmorate.exception.DatabaseException;
+import ru.yandex.practicum.filmorate.exception.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Collection;
+
+import static ru.yandex.practicum.filmorate.validation.PositivityChecker.checkForPositivity;
+import static ru.yandex.practicum.filmorate.validation.PositivityChecker.checkForPositivityCount;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DbFilmService implements FilmServiceInt {
+    private final LocalDate startDate = LocalDate.of(1895, Month.DECEMBER, 28);
+    private final DaoFilmRepository daoFilmRepository;
+    private final DaoLikeRepository daoLikeRepository;
```

Comment:
Если класс реализует какой-либо интерфейс, то при инициализации данного класса в другом, нужно обозначать интерфейс, который он наследует:
```suggestion
    private final DaoLike daoLikeRepository;
```
А спринг уже автоматически выберет и подставит нужную нам реализацию. Таким образом, мы не сломаем программу, если одну из реализаций нужно будет удалить или дополнить

Reply:
Поправь, пожалуйста, по всему проекту